config/mongodb.go

```diff
@@ -23,6 +24,10 @@ func ReadMongoDBConfig() *MongoDBConfig {
 }
 
 func getEnvVariableAsString(name string) string {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
```

Comment:
Just to be sure. Did you check that this really works as expected?

Reply:
When .env is not present: 

![image](https://user-images.githubusercontent.com/18646881/142728247-66f636f0-64c3-4d66-9052-2263357f9285.png)